.vscode/extensions.json

```
@@ -1,3 +1,3 @@
 {
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "maty.vscode-mocha-sidebar"]
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"]
```

```suggestion
  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode", "vitest.explorer"]
```

The Vitest extension works well and is maintained by the Vitest team, let's swap it in here?